src/robotka.h

```diff
@@ -184,7 +186,145 @@ void rkMotorsSetSpeedRight(int8_t speed);
  * \param id číslo motoru od 1 do 4 včetně
  * \param speed rychlost motoru od od -100% do 100%
  */
-void rkMotorsSetSpeedById(int id, int8_t speed);
+void rkMotorsSetSpeedById(uint8_t id, int8_t speed);
+
+/**
+ * \brief Popojetí oběma motory o vzdálenost v mm (blokující).
+ *
+ * Tato funkce zablokuje program, dokud robot danou vzdálenost neujede.
+ *
+ * \param mmLeft kolik milimetrů má ujet levý motor
+ * \param mmRight kolik milimetrů má ujet pravý motor
+ * \param speed rychlost, kterou má robot jet od -100% do 100%
+ */
+void rkMotorsDrive(float mmLeft, float mmRight, uint8_t speed);
+
+/**
+ * \brief Popojetí levým motorem o vzdálenost v mm (blokující).
```

Comment:
O relativní vzdálenost? Od současné pozice do `mmLeft` nebo od počátku měření ujeté vzdálenosti do `mmLeft`?

Reply:
To by bylo na vzdálenost, "popojeď o absolutní vzálenost" nedává smysl.